pkg/machineconfigpools/machineconfigpools.go

```diff
@@ -33,3 +36,18 @@ func GetNodeGroupsMCPs(ctx context.Context, cli client.Client, nodeGroups []nrop
 	}
 	return mcpfind.NodeGroupsMCPs(mcps, nodeGroups)
 }
+
+func GetNodeListFromMachineConfigPool(aContext context.Context, aClient client.Client, mcp mcov1.MachineConfigPool) ([]corev1.Node, error) {
```

Comment:
nice! (spotted `aC++ism` though ;) )

Reply:
yeah ... I do not like it either ... I just came to it because the logic names "context" and "client" were package names in the scope and it seemed hard to read to me.

Reply:
common abbreviations are `ctx` for context (in general) and `cli` for client (in this project but quite often in general)